llvm/test/tools/llvm-exegesis/RISCV/serialize-obj-file.test

```diff
@@ -0,0 +1,33 @@
+# RUN: llvm-exegesis -mtriple=riscv64 -mcpu=sifive-p470 --opcode-name=SH3ADD --benchmark-phase=assemble-measured-code --mode=latency --benchmarks-file=%t.yaml
```

Comment:
I'm thinking we might want to add a new flag that controls the emission of `object_file`? I would prefer not to have it when messing around with exegesis.

I've been wanting to do the same thing with the `assembled_snippet` output for a while given I don't find it particularly useful and it reduces performance (another round trip through the MCJIT/assembler), but that's a separate thing.

Reply:
> I'm thinking we might want to add a new flag that controls the emission of object_file? I would prefer not to have it when messing around with exegesis.

Previously I turned the serialization off if the user is running end-to-end measurement, but I agree that we should make the serialization optional. A new flag has been added

> I've been wanting to do the same thing with the assembled_snippet output for a while given I don't find it particularly useful

It's useful to check if I generated the desired shape of snippets when it showed up in the inconsistency reports. But maybe that's just for RISC-V where ill-formed snippets are more common.